modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/bookmark/BookmarksPage.kt

```diff
@@ -177,9 +177,10 @@ private fun BookmarksView(
             )
         }
         items(state.bookmarks, key = { it }) { bookmark ->
+            val episode = state.episodes.firstOrNull { bookmark.episodeUuid == it.uuid }
```

Comment:
I'm a bit worried by performance here. We iterate over episodes in each composition and in the `items` loop. Would it make sense to have the state hold `Map<String, BaseEpisode>` instead of a `List<BaseEpisode>`. This way lookups by ID will be much faster.

Reply:
I agree, addressed it here: https://github.com/Automattic/pocket-casts-android/pull/2026/commits/be82b2638589b0466c364a541a17e5e751139cde